docs/vertiq_testing_tool/vertiq_testing_tool_start_guide.rst

```diff
@@ -0,0 +1,142 @@
+
+.. warning::
+    Please remove all propellers from any module you plan on testing. Failure to do so can result in harm to you or others around you. Further, please ensure that your module is secured to a stationary platform or surface before attempting to spin it. 
+
+Set the 'Target Voltage' to 3.00 and click 'Send Command'. The module will ramp up to 3 volts over the next 5 seconds specified by the 'Ramp Time' value.
+After 5 seconds, the module should now have a constant, 3V, drive voltage. We can confirm this by plotting the drive volts.
+
+.. image:: ../_static/vertiq_testing_tool_pictures/drive_volts_metric_plotting_example.png
+
+In order to visualize the module's drive voltage, navigate to the widget labeled 'Select a metric to plot'. The upper dropdown menu allows you to pick a client, and the lower a client entry.
+Please refer to :ref:`Getting Started with Vertiq's APIs <getting_started_with_apis>` to learn more about IQUART clients.
+
+In the upper dropdown menu, select *brushless_drive*. This corresponds to the :ref:`Brushless Drive <brushless_drive>` client.
+In the lower dropdown menu, select *drive_volts*, which corresponds with brushless drive's *drive_volts* entry.
+Now, click 'Update Plot'. 
+
+Notice that the *drive_volts* metric is added to the 'Active metrics' widget. By default, *obs_velocity*, *watts*, and *amps* are preloaded in the 'Active metrics' widget.
+Selecting a metric activates it, and updates the live plot with the now active metric. Only one metric can be active at a time, but there is no limit to the number of metrics that can be added to the 'Active metrics' widget.
+
+.. image:: ../_static/vertiq_testing_tool_pictures/remove_metric_example.png
+To remove a metric from the 'Active metrics' widget, simply right click on it and select 'Remove metric'.
+
+
+===============================
+Auto Command and Ramp to Target
+===============================
+.. image:: ../_static/vertiq_testing_tool_pictures/voltage_input.png
+
+Another useful feature in the Voltage, Velocity, and PWM Input tabs is the 'Ramp to target' option found in the Control Panel.
+This feature is enabled by default, which means when you click 'Send Command' after setting the desired Target Voltage, the module ramps to the target voltage incrementally over the time defined in Ramp Time. 
+The increments are determined by the ramp time, so increasing the ramp time will make it so that the module takes longer to reach the target voltage.
+
+The 'Auto Command' feature is disabled by default. When enabled, you will not have to click the 'Send Command' button to send a voltage command. 
+Instead, the application will automatically send the command whenever the target voltage value changes.
+
+Be very cautious when 'Auto Command' is enabled and 'Ramp to Target' is disabled. It is recommended to lower the 'Step Amount' to avoid large jumps between each target voltage command.
```

Comment:
May want to bold this, or put it in a warning box or something so it stands out to people as a thing not to.